profilehooks.py

```diff
@@ -682,7 +683,10 @@ def __str__(self):
         return ''.join(lines)
 
 
-def timecall(fn=None, immediate=True, timer=None):
+def timecall(
+        fn=None, immediate=True, timer=None,
+        log_name=None, log_level=logging.DEBUG
+):
```

Comment:
I've a personal rule of thumb that trailing commas are required whenever the closing bracket is on the next line.  (Someday I'd have to review PEP-8 and see if it's there, or if I came up with it myself.)

In other words, I'd prefer this to read

```python
def timecall(
        fn=None, immediate=True, timer=None,
        log_name=None, log_level=logging.DEBUG,
):
```
or
```python
def timecall(fn=None, immediate=True, timer=None,
             log_name=None, log_level=logging.DEBUG):
```